srcpkgs/python-orocos-kdl/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'python-orocos-kdl'
+pkgname=python-orocos-kdl
+version=1.4.0
+revision=1
+build_style=cmake
+makedepends="python-sip-devel python-devel orocos-kdl eigen3.2"
+short_desc="Python 2 Bindings for Orocos Kinematics and Dynamics C++ library"
```

Comment:
I'd rather not add a python2 library, is it really required ?

Reply:
I can eliminate it. When I asked on #voidlinux recently, the advice was to create both if possible.

Reply:
I wanna avoid adding new python2 stuff